packages/react-reconciler/src/__tests__/ReactSuspenseWithNoopRenderer-test.js

```diff
@@ -3934,19 +3899,21 @@ describe('ReactSuspenseWithNoopRenderer', () => {
         </>,
       );
 
-      // Commit the placeholder
-      Scheduler.unstable_advanceTime(20000);
-      await advanceTimers(20000);
-
-      expect(root).toMatchRenderedOutput(
-        <>
-          <span hidden={true} prop="A1" />
-          <span prop="Loading..." />
-          <span hidden={true} prop="B" />
-          <span prop="Loading..." />
-        </>,
-      );
```

Comment:
Why was this test ending with a placeholder before instead or resolving the values like we do now?

Reply:
This is a regression test that, at the time, triggered a very specific bug in the implementation. It's not really possible to trigger the exact scenario anymore because the semantics of placeholders have changed. But I generally like to keep around old regression tests because they cover cases that we often wouldn't think to write tests for otherwise. It's  not  uncommon for  them to catch things during  refactors, even when the initial bug is no longer relevant.

Reply:
I guess what I'm asking is do you know why it stopped with rendering a fallback?

Was that that regression we were testing?